packages/react-searchbox/examples/with-distinct-field-prop/src/App.js

```diff
@@ -0,0 +1,227 @@
+import React from 'react';
```

Comment:
@anik-ghosh-au7 Example is not meaningful. Remove the author filter and follow the `native` example.